integration-libs/s4-service/order/components/cancel-service-order-headline/cancel-service-order-headline.component.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CartOutlets } from '@spartacus/cart/base/root';
+import { OrderDetailsService } from '@spartacus/order/components';
+import { Observable, map } from 'rxjs';
```

Comment:
`major:` `map` operator should be imported from `rxjs/operators`. It will also solve the problem with types in `order$` Observable.